### PR TITLE
docs(README): add part about test sibling pkg

### PR DESCRIPTION
**Pull request by landonreed**
I was struggling to test a sibling package and this seems like a good way to handle. If someone else has a better way, I'm happy to modify this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,29 @@ See:
  yarn dev # (opens storybook to running component library on localhost:5555)
 ```
 
+## Development
+
+Some packages in otp-ui depend on sibling packages (e.g., `@opentripplanner/core-utils` is used by many of its siblings). In order to test a package with local changes you have made to its sibling, you can run the following find/replace operations to make sure you're depending on your latest work (and not the released version):
+
+1. Find and replace (package-to-test being the package with local changes -- make sure these are committed to first to avoid the find/replace operations below polluting your work):
```
Comment (binh-dam-ibigroup) [inline README.md:25]: This needs to say it is for `package.json`

File: README.md [hunk 2]
```diff
@@ -18,6 +18,29 @@ See:
...
+
+    > "@opentripplanner/package-to-test": **"current-version"**
+    
+    with
+    
+    > "@opentripplanner/core-utils": **"file:../package-to-test"**
+
+2. Find and replace:
```
Comment (binh-dam-ibigroup) [inline README.md:33]: "In your source files, find and replace:"

File: README.md [hunk 3]
```diff
@@ -18,6 +18,29 @@ See:
...
+
+Some packages in otp-ui depend on sibling packages (e.g., `@opentripplanner/core-utils` is used by many of its siblings). In order to test a package with local changes you have made to its sibling, you can run the following find/replace operations to make sure you're depending on your latest work (and not the released version):
+
+1. Find and replace (package-to-test being the package with local changes -- make sure these are committed to first to avoid the find/replace operations below polluting your work):
+
+    > "@opentripplanner/package-to-test": **"current-version"**
+    
+    with
+    
+    > "@opentripplanner/core-utils": **"file:../package-to-test"**
```
Comment (binh-dam-ibigroup) [inline README.md:31]: /package-to-test (not /core-utils)

**Comment by landonreed** [comment]
Thanks @binh-dam-ibigroup. How is that?

File: README.md (hunk 2 above)
Comment (binh-dam-ibigroup) [reply README.md]: Second bullet point still needs a clarification.

**Comment by landonreed** [comment]
@binh-dam-ibigroup, you may have been filtering by only the first commit. I think the second bullet reflects your requested change.

**Comment by binh-dam-ibigroup** [comment]
Sorry, you were right @landonreed.

**Pull request merged**